build: do not strip binaries during 'make install'
--- a/configure
+++ b/configure
@@ -1124,12 +1124,12 @@
 
 .PHONY: install* uninstall*
 install-bin:
-	./install-sh -c -s \$(TARGET) \$(DESTDIR)\$(PREFIX)/bin/\$(TARGET)
+	./install-sh -c \$(TARGET) \$(DESTDIR)\$(PREFIX)/bin/\$(TARGET)
 install: install-bin install-tools
 install-tools: install-tools-yes
 install-tools-yes: \$(TOOLS)
 	mkdir -p \$(DESTDIR)\$(PREFIX)/bin/
-	\$(foreach file, \$(TOOLS), (./install-sh -c -s \$(file) \$(DESTDIR)\$(PREFIX)/bin/ || true);)
+	\$(foreach file, \$(TOOLS), (./install-sh -c \$(file) \$(DESTDIR)\$(PREFIX)/bin/ || true);)
 uninstall:
 	\$(RM) \$(addprefix \$(DESTDIR)\$(PREFIX)/bin/, \$(TARGET) \$(TOOLBINS))
 _EOF
