diff --git a/NsaReader.cpp b/NsaReader.cpp
index 61eb8c2..0540f72 100644
--- a/NsaReader.cpp
+++ b/NsaReader.cpp
@@ -300,6 +300,12 @@ size_t NsaReader::getFile( const char *file_name, unsigned char *buffer, int *lo
 
 struct NsaReader::FileInfo NsaReader::getFileByIndex( unsigned int index )
 {
+    if ( archive_info.fi_list == NULL ) {
+        fprintf( stderr, "NsaReader::getFileByIndex  archive_info.fi_list was NULL!\n" );
+        struct NsaReader::FileInfo bad;
+        return bad;
+    }
+
     int i;
     
     for ( i=0 ; i<num_of_ns2_archives ; i++ ){
diff --git a/SarReader.cpp b/SarReader.cpp
index a1e7365..6afe4bb 100644
--- a/SarReader.cpp
+++ b/SarReader.cpp
@@ -47,6 +47,8 @@ SarReader::~SarReader()
 
 int SarReader::open( const char *name )
 {
+	if (name == NULL) return -1;
+
     ArchiveInfo* info = new ArchiveInfo();
 
     if ( (info->file_handle = fopen( name, "rb" ) ) == NULL ){
@@ -497,6 +499,11 @@ size_t SarReader::getFile( const char *file_name, unsigned char *buf, int *locat
 struct SarReader::FileInfo SarReader::getFileByIndex( unsigned int index )
 {
     ArchiveInfo *info = archive_info.next;
+    if ( info == NULL ) {
+        fprintf( stderr, "SarReader::getFileByIndex  archive_info was NULL!\n" );
+        struct SarReader::FileInfo bad;
+        return bad;
+    }
     for ( int i=0 ; i<num_of_sar_archives ; i++ ){
         if ( index < info->num_of_files ) return info->fi_list[index];
         index -= info->num_of_files;
